docs/eigenlayer/operator-guides/operator-installation.md

```diff
@@ -241,41 +241,56 @@ Step 2: Send **at least 1 Holesky ETH** to the “address” field referenced in
 When using Github for hosting this file please ensure you link to the raw file ([example](https://raw.githubusercontent.com/Layr-Labs/eigenlayer-cli/master/pkg/operator/config/metadata-example.json)), rather than the github repo URL ([example](https://github.com/Layr-Labs/eigenlayer-cli/blob/master/pkg/operator/config/metadata-example.json)).
 :::
 
+**Step 3:** Upload operator image and ensure it is publicly accessible:
+
+You are also required to upload the image of the operator to a publicly accessible location and set the url in your `metadata.json` file. Operator registration only supports `.png` images for now.
+
+**Step 4:** Configure RPC Node:  
+
 The EigenLayer CLI requires access to an Ethereum RPC node in order to post registration. Please plan to either leverage an RPC node provider or run your own local RPC node to reference in operator-config.yaml.
 
-An example list of providers is [available here](https://www.alchemy.com/list-of/rpc-node-providers-on-ethereum) and [here](https://chainlist.org/chain/17000) for your reference.
+Please find example lists of RPC node providers here:
+- https://chainlist.org/
+- https://www.alchemy.com/list-of/rpc-node-providers-on-ethereum
 
 
-#### Holesky Smart contract addresses
+Ensure that your Operator server can reach your RPC provider at this point. You may run the following command from your Operator server:
+`curl -I [your_server_url]`
 
-For operator registration in a Holesky environment, you need to set the DelegationManager contract address as follows:
 
-[Current Testnet Deployment](Current Testnet Deployment): Holesky DelegationManager: [`0xA44151489861Fe9e3055d95adC98FbD462B948e7`](https://holesky.etherscan.io/address/0xA44151489861Fe9e3055d95adC98FbD462B948e7).
 
-```
-# EigenLayer Delegation Manager contract address
-# This will be provided by EigenLayer team
-el_delegation_manager_address: 0xA44151489861Fe9e3055d95adC98FbD462B948e7
-```
 
-#### Registration commands
+**Step 5:** DelegationManager Contract Address
+
+You must configure the correct DelegationManager contract address for your environment.
+- Navigate to [EigenLayer Contracts: Deployments](https://github.com/Layr-Labs/eigenlayer-contracts?tab=readme-ov-file#deployments) and locate the Proxy address for `DelegationManager` for your environment (Mainnet, Testnet).
+- Set the value for `el_delegation_manager_address` in your operator config file to the address for your environment.
+
+
+**Optional:** Set Delegation Approver
+
+Operators have the option to set [delegationApprover](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/src/contracts/interfaces/IDelegationManager.sol#L30) when they register. This is address is used to approve delegation requests from stakers. The delegationApprover address will be required sign and approve new delegation from Stakers to this Operator. Otherwise, the delegationApprover will not be used if the default value is left as `0x000..`.
```

Comment:
A nitpick but I see you've annotated `el_delegation_manager_address` to be a variable but not `delegationApprover`

Reply:
Good catch! Appreciate it. Little things like that help improve the quality of the docs. Updated now